package.json

```diff
@@ -51,6 +51,7 @@
   },
   "homepage": "https://carbon.sage.com",
   "peerDependencies": {
+    "@sage/design-tokens": "^2.3.0",
```

Comment:
I think this should be a breaking change if we're introducing this as a new peer dependency and removing it from the built in dependencies.